e2e/src/usecases/ChooseYourAdventure.js

```diff
@@ -14,20 +14,19 @@ export default ChooseYourAdventure = () => {
     await quickOnboarding({ stopOnCYA: true, cloudBackupEnabled: true })
   })
 
-  it('learn about celo navigates to celo token details page', async () => {
-    await element(by.text('Learn about Celo')).tap()
+  it('learn about valora points navigates to valora points journey page', async () => {
```

Comment:
```suggestion
  it('learn about points navigates to points journey page', async () => {
```
(we want to decrease the use of "valora" in the codebase)

Reply:
@kathaypacific I can still check for `await element(by.text('Learn about Valora Points')).tap()` in the next line, right?
Or do I need to implement some other type of check to exclude "Valora" completely?

Reply:
oh yeah i was thinking about this, i think the assertion is okay (e.g. if we were to run this test for a MS app, we'd need to update this assertion because the test will fail so we won't forget this step, but we may forget to update the test name)